doc/install.rst

```diff
@@ -5,10 +5,10 @@ Installation
 Check dependencies
 ------------------
 You only need ``NumPy``, ``SciPy`` for the package and additionally ``scikit-learn``
-if you would like to run the examples. We recommend
-``Python >= 2.6``,
-``scikit-learn >= 0.17``,
-``NumPy >= 1.6.1`` and ``SciPy >= 0.14``.
+if you would like to run the examples. We currently support
+``Python 2.7``, ``Python 3.5``
+``scikit-learn >= 0.18``,
```

Comment:
+1. The `scikit-learn` API really changed in 0.18 and I think it's reasonable to expect people to upgrade it